spec/features/user_views_authors_list_page_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe "when user visit authors list page" do
+
+  it "sees authors full names, links to their pages and biography" do
+    authors = create_list :author, 2
+    visit authors_path
+
+    authors.each do |author|
+      expect(page).to have_css('.name', author.full_name)
+      expect(page).to have_link(author.full_name, author_path(author))
+      expect(page). to have_css('.biography', author.biography)
+    end
+  end
+
+  it "sees 5 last books images" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.